INSTALL.md

```diff
@@ -70,6 +84,18 @@ Note: On unsupported versions of macOS (more than three releases old),
 Homebrew installs from source, which takes additional time and disk space
 for the `ghc` compiler and dependent Haskell libraries.
 
+Or, you can install pandoc using a [Conda
+forge](https://anaconda.org/conda-forge/pandoc) tool, like
+[`conda`](https://conda.pydata.org/docs/intro.html),
+[`[micro]mamba`](https://mamba.readthedocs.io/en/latest/index.html) or
+[`pixi`](https://prefix.dev). Conda forge contains multiple LaTeX and other
+packages (including `pandoc-citeproc`, `pandoc-plot`, `rsvg-convert` via
+`librsvg` etc.). Note this is a statically-linked executable:
+
+    conda install -c conda-forge pandoc
+    pixi global install pandoc
+    micromamba install pandoc
+
```

Comment:
Rather than repeating this three times, maybe we should have a separate section on installing via Conda.  These repeated paragraphs could then be replaced by something like:
```
Or, you can install pandoc using a [Conda forge][Installing via Conda].
```

Reply:
Done…